pro/src/new_components/BannerOneYear/BannerOneYear.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+import { ReactComponent as Banner } from './assets/one_year_banner.svg'
+import { ReactComponent as CloseButton } from './assets/close.svg'
+import styles from './BannerOneYear.module.scss'
+
+const BannerOneYear = () => {
+
+  const [isCloseForEver, setIsCloseForEver] = useState(localStorage.getItem('iscloseForEver'));
+    const closeForEver = () => {
+      localStorage.setItem('iscloseForEver', 'true');
+      setIsCloseForEver(localStorage.getItem('iscloseForEver'))
```

Comment:
Ici c'est un détail mais il ne me semble pas utile d'appeller `localStorage.getItem('iscloseForEver')`  sachant qu'on vient de le passer à `true`

```suggestion
      setIsCloseForEver(true)
```